R/fread.R

```diff
@@ -81,6 +81,16 @@ fread <- function(input="",file,sep="auto",sep2="auto",dec=".",quote="\"",nrows=
   stopifnot(length(skip)==1L, !is.na(skip), is.character(skip) || is.numeric(skip))
   if (skip=="__auto__") skip=-1L   # skip="string" so long as "string" is not "__auto__". Best conveys to user something is automatic there (than -1 or NA).
   if (is.double(skip)) skip = as.integer(skip)
+  stopifnot(is.null(na.strings) || is.character(na.strings))
+  tt = grep("^ +$", na.strings)
+  if (length(tt)) {
+    warning('na.strings contains some blank strings. They have been removed. strip.white=TRUE (default) together with na.strings="" turns any number of spaces into NA in string columns')
+    na.strings = na.strings[-tt]
```

Comment:
Should this be 

```r
tt = grep("^\\s+$", na.strings)
if (strip.white && length(tt)) {
  warning('na.strings contains strings consisting only of whitespace, yet strip.white=TRUE. These will be ignored: strip.white=TRUE (default) together with na.strings="" turns any number of spaces into NA in string columns'.)
}
```

*i.e.* not just repeated spaces and not 'blank' strings but 'whitespace-only' strings.

Reply:
Well spotted. Yes, will do.